resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,205 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "Renkforce RF100",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Simon Peter (based on RF100.ini by Conrad Electronic SE)",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "manufacturer": "Renkforce",
+        "visible": true
+    },
+
+    "overrides": {
+        "adhesion_type": {
+            "value": "skirt"
+        },
+        "bottom_thickness": {
+            "value": "0.5"
+        },
+        "brim_line_count": {
+            "value": "20.0"
+        },
+        "cool_fan_enabled": {
+            "value": "True"
+        },
+        "cool_fan_full_at_height": {
+            "value": "0.5"
+        },
+        "cool_fan_speed_max": {
+            "value": "100.0"
+        },
+        "cool_fan_speed_min": {
+            "value": "100.0"
+        },
+        "cool_lift_head": {
+            "value": "True"
+        },
+        "cool_min_layer_time": {
+            "value": "5.0"
+        },
+        "cool_min_speed": {
+            "value": "10.0"
+        },
+        "infill_before_walls": {
+            "value": "True"
+        },
+        "infill_overlap": {
+            "value": "15.0"
+        },
+        "layer_0_z_overlap": {
+            "value": "0.22"
+        },
+        "layer_height_0": {
+            "value": "0.3"
+        },
+        "machine_depth": {
+            "value": "100"
+        },
+        "machine_end_gcode": {
+            "value": "';End GCode\nM104 S0 ;extruder heater off\nM140 S0 ;heated bed heater off (if you have it)\nG91 ;relative positioning\nG1 E-1 F300 ;retract the filament a bit before lifting the nozzle, to release some of the pressure\nG1 Z+0.5 E-5 X-20 Y-20 F{speed_travel} ;move Z up a bit and retract filament even more\nG28 X0 Y0 ;move X/Y to min endstops, so the head is out of the way\nM84 ;steppers off\nG90 ;absolute positioning'"
```

Comment:
This line still fails to parse, since `\n` already gets unescaped when the string is read from the JSON, and then it evaluates a multi-line string which has a syntax error:

    2017-02-07 11:57:43,968 - ERROR - UM.Settings.SettingFunction.__init__ [36]: Parse error in function (';Sliced at: {day} {date} {time}
    G21 ;metric values
    G90 ;absolute positioning
    M82 ;set extruder to absolute mode
    M107 ;start with the fan off
    G28 X0 Y0 ;move X/Y to min endstops
    G28 Z0 ;move Z to min endstops
    G1 Z15.0 F{speed_travel} ;move the platform down 15mm
    G92 E0 ;zero the extruded length
    G1 F200 E3 ;extrude 3mm of feed stock
    G92 E0 ;zero the extruded length again
    G1 F{speed_travel}
    M117 Printing...') for setting: EOL while scanning string literal (eval, line 1)

It fails because it tries this:

    eval("';End GCode
    M104 S0 ;extruder heater off
    <...>
    G90 ;absolute positioning'")

That is a syntax error. The same happens with the `machine_start_gcode` setting.